resources/assets/uad_lists.json

```
@@ -6248,7 +6248,7 @@
   {
     "id": "com.sec.android.app.vepreload",
     "list": "Oem",
-    "description": "Samsung video editor\nLets you add add transitions, music, stickers and text to your videos. You can also change the speed of the action, \nor even add filters to switch up the mood of your videos.\n",
+    "description": "Samsung video editor\nLet's you add add transitions, music, stickers and text to your videos. You can also change the speed of the action, \nor even add filters to switch up the mood of your videos.\n",
```

```suggestion
    "description": "Samsung video editor\nLets you add add transitions, music, stickers and text to your videos. You can also change the speed of the action, \nor even add filters to switch up the mood of your videos.\n",
```
Incorrect. "Lets" is correct: https://www.grammarly.com/blog/lets-vs-lets/#:~:text=The%20difference%20between%20let's%20and%20lets&text=Let's%20is%20a%20contraction%20of%20%E2%80%9Clet%20us.%E2%80%9D%20You%20use,to%20allow%20or%20give%20permission.